src/main/flight/pid.h

```diff
@@ -74,11 +74,11 @@ typedef struct pid8_s {
 typedef struct pidProfile_s {
```

Comment:
@martinbudden without updating PG version this change will corrupt configuration when updating the firmware